packages/preset-mini/src/_variants/startingstyle.ts

```diff
@@ -0,0 +1,17 @@
+import type { Variant } from '@unocss/core'
+
+export const variantStartingStyle: Variant = {
+  name: 'starting',
```

Comment:
Could we have some test to snapshot it's behavior? Thanks

Reply:
Done!